src/library_wasmfs.js

```diff
@@ -107,9 +107,12 @@ mergeInto(LibraryManager.library, {
       // TODO: Remove dependency on FS.cwd().
       // User code should not be using FS.cwd().
       // For file preloading, cwd should be '/' to begin with.
-      return '/';
+      return withStackSave(() => {
```

Comment:
You probably want to do something like `var ptr = stackAlloc({{{ PATH_MAX }}})` and then pass the result to `__wasmfs_getcwd` (as pointer + length), and then `return UTF8ToString(ptr);`.

Reply:
Will rectify this. Thank you